Source/Tools/LinuxEmulation/LinuxSyscalls/Syscalls/Thread.cpp

```diff
@@ -463,6 +463,7 @@ void RegisterThread(FEX::HLE::SyscallHandler* Handler) {
     ThreadObject->StatusCode = status;
 
     FEX::HLE::_SyscallHandler->UninstallTLSState(ThreadObject);
+    FEX::HLE::_SyscallHandler->FlushAndCloseCodeMap();
```

Comment:
`exit` only exits the current thread, so this needs to be moved to e.g. `ThreadManager::DestroyThread`, where we can check if this is the last remaining thread.